SamsungPay/src/main/java/com/braintreepayments/api/SamsungPayInternalClient.java

```diff
@@ -151,7 +151,7 @@ public void onSuccess(CustomSheetPaymentInfo customSheetPaymentInfo, String s, B
             @Override
             public void onFailure(int errorCode, Bundle bundle) {
                 if (errorCode == SpaySdk.ERROR_USER_CANCELED) {
-                    UserCanceledException userCanceledError = new UserCanceledException("User canceled Samsung Pay.");
+                    UserCanceledException userCanceledError = new UserCanceledException("User canceled Samsung Pay.", true);
```

Comment:
This seems like an explicit cancellation from the SDK, but we don't have a Samsung device to confirm this on - can anyone assist here?

Reply:
Yeah I can take a look I have a Samsung.

Reply:
This lines up! There's a "Cancel" button and the system back button – both result in the same `SpaySdk.ERROR_USER_CANCELED` error code on a Galaxy S21.